.github/workflows/quick_pr.yml

```diff
@@ -43,7 +49,8 @@ jobs:
                 new_issue_url=$(gh issue create \
                     --title "$TITLE" \
                     --assignee "$AUTHOR" \
```

Comment:
i think a comment would be nice. Maybe also state why the link to the project is necessary here.